### PR TITLE
Adjusting your jumpsuit now properly exposes your chest and/or arms as applicable.

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/5932.

As pointed in that issue report, some jumpsuits expose only the arms, some expose arms and chest. (please don't make any adjusted jumpsuits that expose any other configuration of parts for the love of fuck accounting for this was snowflakey enough) Jumpsuits have a boolean var to track this. It is set to 0 (exposes both) by default. If I missed any, let me know.

### CODE REVIEW

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -229,6 +229,7 @@ BLIND     // can't see anything
 	var/sensor_mode = 0	/* 1 = Report living/dead, 2 = Report detailed damages, 3 = Report location */
 	var/can_adjust = 1
 	var/adjusted = 0
+	var/modesty = 0 // 0 = exposes chest and arms, 1 = exposes arms only
```
Comment (duncathan) [inline code/modules/clothing/clothing.dm]: Make them defines

Comment (bgobandit) [inline code/modules/clothing/clothing.dm]: I've looked at all the defines in the code and I'm not seeing what to do here

Comment (MrStonedOne) [inline code/modules/clothing/clothing.dm]: Defines isn't right here. we don't use defines for bool vars.

the var name should be changed to altCoversChest alt_covers_chest

**Comment by freerealestate** [comment]
neat :+1:

**Pull request merged**